lib/features/auto-subscribe.js

```diff
@@ -60,6 +87,8 @@ module.exports = ({
           }
         });
 
+        _listenOnSyncEvents({rtm, filterFn});
+
         return Promise.all(promisedSubscriptions);
```

Comment:
The whole block is nested to deep for my taste and is are really bad example of callback hell with promises. Maybe take the chance and improve it?